plugins/BEdita/Core/src/Model/Table/UsersTable.php

```diff
@@ -292,9 +292,16 @@ protected function findExternalAuth(Query $query, array $options = [])
         return $query->innerJoinWith('ExternalAuth', function (Query $query) use ($options) {
             $query = $query->find('authProvider', $options);
             if (!empty($options['username'])) {
-                $query = $query->where([
-                    $this->ExternalAuth->aliasField('provider_username') => $options['username'],
-                ]);
+                if (is_array($options['username'])) {
+                    $query = $query->where(fn(QueryExpression $exp) => $exp->in(
+                        $this->ExternalAuth->aliasField('provider_username'),
+                        $options['username']
+                    ));
+                } else {
+                    $query = $query->where([
+                        $this->ExternalAuth->aliasField('provider_username') => $options['username'],
+                    ]);
+                }
```

Comment:
There's nothing wrong in using `IN` for a single element, in case you wish to make this slightly simpler.

```suggestion
                $query = $query->where(fn (QueryExpression $exp) => $exp->in(
                    $this->ExternalAuth->aliasField('provider_username'),
                    (array)$options['username'],
                ));
```